docs/api/topics/firefox_os_addon_submission.rst

```diff
@@ -94,6 +91,28 @@ Add-on Creation
 
     **Response**
 
-    An :ref:`add-on <addon-response-label>`.
+    The newly created :ref:`add-on <addon-response-label>`. Note that an add-on
+    version is created automatically for you.
 
     :status: 201 successfully created.
+
+Add-on Version Creation
```

Comment:
I'm calling them `Firefox OS Add-on` in all page headers, but that was a bit too much redundancy to do that everywhere.